src/vulkan-renderer/wrapper/vma.cpp

```diff
@@ -0,0 +1,78 @@
+#include "inexor/vulkan-renderer/wrapper/vma.hpp"
+
+#define VMA_IMPLEMENTATION
+
+// It makes memory of all new allocations initialized to bit pattern 0xDCDCDCDC.
+// Before an allocation is destroyed, its memory is filled with bit pattern 0xEFEFEFEF.
+// Memory is automatically mapped and unmapped if necessary.
+#define VMA_DEBUG_INITIALIZE_ALLOCATIONS 1
+
+// Enforce specified number of bytes as a margin before and after every allocation.
+#define VMA_DEBUG_MARGIN 16
+
+// Enable validation of contents of the margins.
+#define VMA_DEBUG_DETECT_CORRUPTION 1
+
+#include <vma/vk_mem_alloc.h>
```

Comment:
I think it would look nicer if the includes went above the defines.

Reply:
I thought that too, but isn't the defines used inside the vma header? (at least i guessed that)

Reply:
I thought that too, but isn't the defines used inside the vma header? (at least i guessed that)

Reply:
That's a good point :D